shipment_planning_maxoptra/models/res_partner.py

```diff
@@ -15,6 +15,16 @@ class ResPartner(models.Model):
         "partner to set on Batch pickings after import.",
     )
 
+    partner_key = fields.Text(string="Key")
+
+    _sql_constraints = [
+        (
+            "partner_key_uniq",
+            "unique(partner_key)",
```

Comment:
Can this constraint be installed if you have null values in the table? :thinking:

Reply:
checked it, afaik null values are not considered in that case so it won't break.

Reply:
I mean, it won't break if the constraint cannot be installed, but if we cannot install this constraint in SQL shouldn't you move it to a python one?